constants.go

```diff
@@ -0,0 +1,49 @@
+package ykoath
```

Comment:
I would prefer the constants to be private tbh - they do help readability but trigger linting advice when undocumented and public.